pkgs/applications/version-management/git-and-tools/git-big-picture/default.nix

```diff
@@ -1,14 +1,14 @@
-{ fetchFromGitHub, python2Packages, lib, stdenv, git, graphviz }:
+{ fetchFromGitHub, python3Packages, lib, stdenv, git, graphviz }:
 
-python2Packages.buildPythonApplication rec {
+python3Packages.buildPythonApplication rec {
   pname = "git-big-picture";
-  version = "0.10.1";
+  version = "1.0.0";
 
   src = fetchFromGitHub {
-    owner = "esc";
+    owner = "git-big-picture";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0b0zdq7d7k7f6p3wwc799347fraphbr20rxd1ysnc4xi1cj4wpmi";
+    sha256 = stdenv.lib.fakeSha256; # TODO
```

Comment:
```suggestion
    sha256 = "14yf71iwgk78nw8w0bpijsnnl4vg3bvxsw3vvypxmbrc1nh0bdha";
```

Reply:
Thanks, I've put it in now to have the CI give it a try. Can you teach me how to derive a hash like that myself for next time using says plain Python on a non-nixOS Linux?